.travis.yml

```diff
@@ -37,14 +37,17 @@ matrix:
       - PRODUCT_TESTS='-g hdfs_impersonation'
       - CLUSTER_TYPE='singlenode-hdfs-impersonation'
     - env:
-      - PRODUCT_TESTS='-g hdfs_impersonation,authorization'
+      - PRODUCT_TESTS='-g hdfs_impersonation,authorization,storage_formats,cli'
       - CLUSTER_TYPE='singlenode-kerberos-hdfs-impersonation'
     - env:
       - PRODUCT_TESTS='-g ldap_cli'
       - CLUSTER_TYPE='singlenode-ldap'
     - env:
       - MVN_TESTS='-Dtest=TestLocalBinarySpilledQueries,TestPostgreSqlDistributedQueries,TestMySqlDistributedQueries'
       - MVN_MODULES='-pl presto-tests,presto-mysql,presto-postgresql'
+    - env:
+      - PRODUCT_TESTS='-g hdfs_no_impersonation,storage_formats,cli'
```

Comment:
We need to test all varieties of authorization -- nothing, hdfs impersonation, kerberos, and kerberos impersonation. That's non-negotiable. And we need to actually test them. I think that 3 seconds vs 2:35 is acceptable.

If startup time is so high, maybe that means we should go back to running multiple product test runs per executor. We need to have different docker containers for all of these environments, unfortunately.